### ISSUE TITLE
Adds "track_errors = On" which is deprecated with PHP 7.2

### ISSUE THREAD

**Issue by mbiebl** [phpmanager/phpmanager#20]
Forwarding this issue from https://github.com/RonaldCarter/PHPManager/issues/12

Hi,

we are using PHP 7.2.7 with IIS/Windows Server 2016.

When using PHP Manager to switch PHP from debug into development mode, it adds the following line to php.ini:
```
track_errors = On
```
With php 7.2 this now generates a deprecation warning:
```
Deprecated: Directive 'track_errors' is deprecated in Unknown on line 0
```
See also http://php.net/manual/en/migration72.deprecated.php

PHP Manager should stop adding/setting track_errors when PHP 7.2 or newer is used.

**Comment by lextm** [comment]
Test installers can be found at https://ci.appveyor.com/project/lextm/phpmanager/build/artifacts

**Comment by mbiebl** [comment]
hi, thanks for the quick response.
I tried the installer from the above link. I had PHP Manager 1.3 installed, the new version upgraded successfully  (The only odd issue being that after the upgrade I still had a C:\Program Files (x86)\PHP Manager 1.3 for IIS 10\ directory containing Web.Management.PHP.PowerShell.InstallState)

Unfortunately this version does not work for me. If I open the "PHP Error Reporting" page, IIS Manager (InetMgr.exe) locks up and uses 50% CPU (on a dual core system). The only thing I can do at this point is to kill the InetMgr.exe process via the task manager.
Other parts of PHP Manager seem to work correctly, e.g. I tested enabling/disabling a extension.

**Comment by lextm** [comment]
`.InstallState` is a leftover of the installer (I don't know how to get rid of such yet).

I am testing daily on a Windows Server 2016 machine, so it would be a little bit strange on the issue you met.

Can you manually remove `track_errors` from `php.ini` and try again?

I also have a new build available for testing as 2.0 RC 2,

https://github.com/phpmanager/phpmanager/releases

**Comment by mbiebl** [comment]
Same problem with the RC2 release.
OS is Windows Server 2016, PHP version  7.2.8 (x86 nts) installed under C:\Program Files (x86)\PHP\v7.2.
Problem is reproducible 100% even after a reboot.
I've attached the php.ini (based on php.ini-production) which triggers the error
[php.ini.txt](https://github.com/phpmanager/phpmanager/files/2280465/php.ini.txt)

**Comment by mbiebl** [comment]
> .InstallState is a leftover of the installer (I don't know how to get rid of such yet).

Should I create a separate issue for that?

**Comment by mbiebl** [comment]
Notice how line 854 in php.ini contains ```track_errors = Off```
I *think* this was added by a previous PHP manager version when I used it to switch into production mode. Removing that line did not help though.

**Comment by lextm** [comment]
@mbiebl The issue is now fixed in RC3.

**Comment by mbiebl** [comment]
Thanks, RC3 works much better, the high CPU usage is gone.
Tested with both v7.0: track_errors is set to On/Off
and with v7.2: track_errors is no longer set.

The only minor issue I noticed: if your old php.ini has ```track_errors = On``` set, PHP manager will not actively remove that deprecated setting when PHP 7.2 is used. This setting needs to be removed manually on upgrades. But I can live with that, as this is a one-time change.
